SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -524,6 +524,11 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 	RenamedStartupActors.Remove(ThisActor->GetFName());
 }
 
+const FString USpatialNetDriver::GetUniqueIdentifier() const
+{
+	return Connection->GetWorkerId();
+}
+
```

Comment:
Might we want to return `Connection->GetWorkerId() + Super::GetUniqueIdentifier()` ?

Reply:
Currently super is `""` - if we are in the future going to add a different unique identifier or change the class ancestry, any problems or further work should be trivial to reconsider at that point.